src/commands/setup/interface.ts

```diff
@@ -1 +1,5 @@
-export type DatadogSite = "US" | "EU" | "US3" | "US5" | "GOV";
+export const datadogSites = ["US", "EU", "US3", "US5", "GOV"] as const;
```

Comment:
`US` and `EU` are deprecated, customers should use `US1` and `EU1` respectively.

```suggestion
export const datadogSites = ["US1", "EU1", "US3", "US5", "GOV"] as const;
```